koans/1.intro.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'tape'
 
 import {
-  arrow,
+  arr,
```

Comment:
tl;dr is: you make a good point, my 2 cents are that it should be full name

1. I don't think we can use Haskell conventions in JS. To start with, type annotations in Flow are closer to the C++ style, as it is for the rest of the JS (C family syntax).
2. Flow's behaviour is indeed inconvenient. Of the top of my head comes the idea of naming the type `ArrowType` but that's ugly af. 
3. I don't know if a Category and a Type class are not related enough to use the same naming conventions. In this I lack knowledge to say, if the constructor that you have is not a canonical constructor then for sure it should _not_ be capitalized
4. I think full names are almost always better =) . In this particular case it can be confused with `array` so I'd strongly recommend it being full name either way.